rest_framework-stubs/response.pyi

```diff
@@ -1,19 +1,23 @@
-from typing import Any, Dict, Mapping, Optional
+from typing import Any, Dict, List, Mapping, Optional, Union
 
 from django.template.response import SimpleTemplateResponse
+from rest_framework.request import Request
 
 class Response(SimpleTemplateResponse):
-    data: Optional[Any] = ...
+    data: Optional[Union[List[Dict[str, Any]], Dict[str, Any]]] = ...
```

Comment:
should be Any, these data just passed to renderer user can define renderer that accepts arbitrary data, also your typing doesn't handle all the cases of valid json, for example we have case where `data: List[List[str]]`

Reply:
Agreed. After restoring decorators logic in #119, many `response` variable types became visible to mypy in the test suite, and this hint causes false positives.

I've reverted it to `Any` in #119.